package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "scripts": {
     "build:js": "esy b dune build test/Test.bc.js",
+    "build:dom": "refmterr dune build ./examples/dom/WebReconciler.bc.js",
```

Comment:
For this to work on Windows - I needed to change it to `esy b refmterr dune build ./examples/dom/WebReconciler.bc.js`